Administracion y footer

CCande12 opened the pull request.

CCande12 committed:
se comienza a realizar el footer, quedo a la espera de la aprobacion

CCande12 committed:
se agrego el logo y se arreglo la paleta de  colores

marukuenneth commented:
para mi esta todo de diez para hacer el merge

pabloLlanes reviewed (commented):
Buenas Chicos, si bien el pr esta bien realizado.
La rama (branch) que se intenta subir, presenta algunos conflictos

CCande12 committed:
Merge branch 'dev' into administracionYFooter

CCande12 committed:
se hace el merge luego de la resolucion de conflictos en la rama del Footer

ibrahimeric committed:
commit

ibrahimeric committed:
Please enter the commit message for your changes. Lines starting
with '#' will be ignored, and an empty message aborts the commit.

On branch administracionYFooter
Your branch is up to date with 'origin/administracionYFooter'.

Changes to be committed:
	renamed:    footer.html -> pages/footer.html

ibrahimeric committed:
Please enter the commit message for your changes. Lines starting
with '#' will be ignored, and an empty message aborts the commit.

On branch administracionYFooter
Your branch is up to date with 'origin/administracionYFooter'.

Changes to be committed:
	renamed:    css/style.css -> css/footer.css
	new file:   index.html
	deleted:    pages/footer.html

ibrahimeric committed:
Please enter the commit message for your changes. Lines starting
with '#' will be ignored, and an empty message aborts the commit.

On branch administracionYFooter
Your branch is up to date with 'origin/administracionYFooter'.

Changes to be committed:
	modified:   index.html
